src/controller/Controller.js

```diff
@@ -0,0 +1,86 @@
+import OutputView from '../view/OutputView.js';
+import InputView from '../view/InputView.js';
+import CarModel from '../model/CarModel.js';
+import { validateName, validateTryCount } from '../validation.js';
+import { getRandomNumber } from '../utils/getRandomNumber.js';
+
+export default class Controller {
+  constructor() {
+    this.inputView = new InputView();
+    this.outputView = new OutputView();
+  }
+
+  async start() {
+    const names = await this.getNames();
+    const count = await this.getCount();
+
+    const cars = this.createCars(names);
+    this.outputView.printEmptyLine();
+    this.outputView.printResult();
+
+    this.runRace(cars, count);
+    this.findWinner(cars);
+  }
+
+  runRace(cars, count) {
+    for (let i = 0; i < count; i++) {
+      for (let j = 0; j < cars.length; j++) {
+        const car = cars[j];
+        this.runSingleRound(car);
+      }
+      this.outputView.printEmptyLine();
+    }
+  }
```

Comment:
다른 분들이 comment를 잘 남겨주신 것 같아요~

```suggestion
runRace(cars, count) {
    for (let i = 0; i < count; i++) {
        cars.forEach(car => this.runSingleRound(car));
        this.outputView.printEmptyLine();
    }
}
```

많은 예시 중 하나지만, 이렇게 `forEach` method를 사용해 볼 수 있겠네용

Reply:
요구사항을 못지켜서 너무 아쉽네요...ㅠㅠ 그래도 덕분에 깔끔한 코드 알아갑니다!!👍